packages/client/components/text-question-answer/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { TextField, Grid, Typography } from '@material-ui/core'
+import styled from 'styled-components'
+
+const Container = styled(Grid)`
+  padding: 8px;
+`
+
+const AnswerField = styled(TextField)`
+  width: 100%;
+`
+
+interface SingleAnswerProps {
+  editable: boolean
+  answer: string | null
+  from: string | false
+}
+
+const SingleAnswer = ({ editable, answer, from }: SingleAnswerProps) => {
+  if (editable) {
+    return <AnswerField multiline variant="outlined" value={answer} />
+  }
+  const author = from ? `${from}: ` : ''
+  return <Typography>{`${author}${answer || '[No response]'}`}</Typography>
```

Comment:
Is there a difference between `answer === ''` and `answer === null`? 
If not, `answer` should probably just be a string.
If so, this should probably be `??` instead of `||`

Reply:
> Is there a difference between `answer === ''` and `answer === null`?
> If not, `answer` should probably just be a string.
> If so, this should probably be `??` instead of `||`

I think answer can be an empty string. If an answer is empty, then frontend should get null instead of empty array.

Reply:
Which one better represents an empty answer? empty string or null?

Reply:
Unless there's a good reason for it to be `null`, I think we should make the types as strict as possible, so we have less type-checking later on.